_sass/main.scss

```diff
@@ -1,14 +1,10 @@
 @import "settings_colors";
 
-@import "vanilla-brochure-theme/scss/theme";
-@include vanilla-brochure-theme;
+@import "vanilla-framework/scss/build";
 
 @import "patterns_strip";
 @include cloud-init-p-strips;
 
-@import "patterns_lists";
-@include cloud-init-p-lists;
-
 @import "patterns_heading-icon";
 @include cloud-init-p-heading-icon;
 
```

Comment:
You can remove everything from this file except the color settings and vanilla import, and then remove the `_patterns` files